modules/core/src/lib/index.js

```diff
@@ -22,7 +22,11 @@
 export {COORDINATE_SYSTEM} from './constants';
 
 // Export core objects
+export {default as Deck} from './deck';
```

Comment:
Total nit: I am not a fan of chasing hierarchical imports through nested `index.js` files. If we don't do the require hacks in the other debug PR, does this file add anything rather than having these inline in the top level export.